app/backend/resources/schema.edn

```diff
@@ -1823,7 +1823,103 @@
   :txes [[{:db/ident :vektorio/granted-projects
            :db/valueType :db.type/string
            :db/cardinality :db.cardinality/many
-           :db/doc "ID-s of the Vektor projects the user has granted access to"}]]}]
+           :db/doc "ID-s of the Vektor projects the user has granted access to"}]]}
+
+ {:db/ident :migration/thk-contracts
+  :txes [[{:db/ident :thk.contract/number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Link to contract file in document management system for procurement part"}
+
+          {:db/ident :thk.contract/procurement-number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Procurement no /Hanke nr - (if starts with number then converted into link to State procurement registry as https://riigihanked.riik.ee/rhr-web/#/procurement/<proc_No>), if starts with other symbol then leave it as a number)"}
+
```

Comment:
Add `procurement-id` from csv as value.

Reply:
marked as `:db/unique :db.unique/value` so we use that as the id

Reply:
should be marked `:db.unique/identity` instead to get the upset behavior